problems/secretz/problem.md

```diff
@@ -8,7 +8,8 @@ process.argv[3] and the cipher initialization vector as process.argv[4].
 You can pass these arguments directly through to `crypto.createDecipheriv()`.
 
 The built-in zlib library you get when you `require('zlib')` has a
-`zlib.createGunzip()` that returns a stream for gunzipping.
+`zlib.createGunzip()` that returns a stream for gunzipping. But you can skip it
```

Comment:
I think we can remove the `zlib` reference since will be unused; and update the `tar` paragraph mentioning that can receive a gzipped file.